### PR TITLE
Redmine tutorial

### PR DESCRIPTION
**Pull request by zjhmale**
redmine 机器人引导

@xtang @yuanbohan @loddit

### CODE REVIEW

**Comment by zjhmale** [comment]
poke

File: robots/redmine.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+```bash
+$ cd $REDMINE_ROOT/plugins #进入redmine安装路径下的plugins路径
+$ git clone git://github.com/suer/redmine_webhook.git
+$ bundle install #安装webhook插件以来的gem
+$ rake redmine:plugins:migrate RAILS_ENV=production
+```
+
+## 进入 Webhook 界面
+
+进入需要加入 Webhook 的项目主页 点击「配置」 在点击「WebHook」
```
Comment (yuanbohan) [inline robots/redmine.md]: 断句有点随意，加上标点吧

**Comment by yuanbohan** [comment]
LGTM

**Comment by zjhmale** [comment]
poke @xtang @loddit

**Comment by xtang** [comment]
LGTM

**Pull request merged**